src/compile/Component.ts

```diff
@@ -51,7 +51,7 @@ export default class Component {
 	instance_scope: Scope;
 	instance_scope_map: WeakMap<Node, Scope>;
 
-	meta: Meta;
+	optionsTag: Options;
```

Comment:
I'm definitely open to a better name than `optionsTag`. `option` was already taken by the options passed directly to the compiler itself.

Reply:
Maybe rename both? We could have `compilerOptions` and `componentOptions`.

Reply:
Went ahead and pushed that change. Alter or revert or whatever as desired.